web/templates/message_view_header.hbs

```diff
@@ -1,5 +1,5 @@
 {{#if stream_settings_link}}
-<a class="stream" href="{{stream_settings_link}}">
+<a class="stream-settings" href="{{stream_settings_link}}">
```

Comment:
Let's name it `message-header-stream-settings-button`.

Reply:
done